app/assets/stylesheets/admin/forms/_forms.scss

```diff
@@ -30,6 +30,13 @@ form {
   }
 }
 
+.checkbox {
```

Comment:
Not sure if there is a better way to deal with this with simple form. Not super familiar with how the bootstrap wrappers work.